### ISSUE TITLE
Stick to a clearly defined directory structure

### ISSUE THREAD

**Issue by ashutoshgngwr** [harshit0209/EasyPermit#2]
Hi! I loved the overall _jugaad_ system. Although I found the directory structure for the project to be somewhat counter-intuitive.

AFAIK, by convention, the library source is placed in a top-level module right alongside the `app` module. The module is also named `library` to establish clear distinction. I have seen some projects renaming `app` module to `demo` or `example` for the same reason as well. It will also help in maintaining both demo and library in the same repository.

```
EasyPermit/
├── example/
│   └── src/
│   └── build.gradle
└── library/
│   └── src/
│   └── build.gradle
└── build.gradle
```

PS, this is a great initiative and it really simplifies the permission request flow.

**Comment by harshit0209** [comment]
Hello! I am glad you loved it. And I am more than happy for this suggestion.
I have made some changes please check.

I would be glad to see your interaction with this repo.
PS. Again thanks for support.

**Comment by ashutoshgngwr** [comment]
Awesome! Resolved in https://github.com/harshit0209/EasyPermit/commit/2fedc795ffdcc2dcb2418114ee2c4fd53563329d.

**Comment by harshit0209** [comment]
> Awesome! Resolved in [2fedc79](https://github.com/harshit0209/EasyPermit/commit/2fedc795ffdcc2dcb2418114ee2c4fd53563329d).

You are great. Thanks for telling me.
Happy Programming!
